### PR TITLE
Change style of add card buttons to be less distracting

### PR DESCRIPTION
**Pull request by jancborchardt**
The full-color add buttons are a bit much when you actually have some cards in the stack, tends to be very distracting. ;) How about this:

Before:
screenshot from 2017-09-29 10-10-39
After:
screenshot from 2017-09-29 10-04-23

Only issue is I can’t seem to get the font-color visible. Any idea @nextcloud/deck?

### CODE REVIEW

**Comment by juliushaertl** [comment]
@jancborchardt Thanks for the PR, looks pretty good to me. 

The font color for the input field is being set by angular here: https://github.com/nextcloud/deck/pull/310/files#diff-a8325f055da9615ba3ebbb5e7da3d790R112 After removing the color from ng-style the rule from the css should be applied.

**Comment by juliushaertl** [comment]
Ah one more thing, maybe we should also add a border radius to the cards then, looks a bit odd now with only the add buttons being rounded.

**Comment by juliushaertl** [comment]
@jancborchardt I've made some changes to this. 

bildschirmfoto vom 2017-10-24 12-33-13

File: css/style.scss [hunk 1]
```diff
@@ -334,7 +334,8 @@ input.input-inline {
 	white-space: normal;
 	position: relative;
 	opacity: 1.0;
-	-webkit-box-shadow: 0 0 5px $color-darkgrey;
+	-webkit-box-shadow: 0 0 3px $color-darkgrey;
```
Comment (pixelipo) [inline css/style.scss]: box-shadow only for legacy webkit broewsers?

**Comment by jancborchardt** [comment]
@juliushaertl looks good to me! :)

**Comment by juliushaertl** [comment]
@pixelipo Thanks, fixed.

**Comment by jancborchardt** [comment]
So, is this good to go? Needs rebase now also ;)

**Comment by juliushaertl** [comment]
Rebased. Let's merge since CI was happy before. 👍 

Thanks @jancborchardt looks much cleaner now

**Comment by pixelipo** [comment]
@juliushaertl sorry I had no time to revisit this PR

**Comment by jancborchardt** [comment]
Weehuu! :) Thanks, and happy to help!

**Pull request merged**